apps/frontend/src/app/2_molecules/PromoCard/PromoCard.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from 'react';
+
+import classNames from 'classnames';
+import { t } from 'i18next';
+
+import { SupportedTokens } from '@sovryn/contracts';
+
+import { getTokenDisplayName } from '../../../constants/tokens';
+import { translations } from '../../../locales/i18n';
+import { AmountRenderer } from '../AmountRenderer/AmountRenderer';
+import { PairRenderer } from '../PairRenderer/PairRenderer';
+import styles from './PromoCard.module.css';
+
+type PromoCardProps = {
+  className?: string;
+  asset1: SupportedTokens;
+  asset2: SupportedTokens;
+  apy: string;
+  rewards: string;
+  rewardToken: SupportedTokens;
+  rewardsLabel: string;
```

Comment:
```suggestion
  apy: string;
  rewards: string;
  rewardToken: SupportedTokens;
  rewardsLabel: string;
```
I don't like the fact that the component is very opinionated - the APY section is basically hardcoded there. I discussed it with @soulBit as well and we agreed on following - pass only labels for right and left content and then respective children, so you'll have something like `label1`, `label2`, `child1`, `child2` (children will be of type `ReactNode).